<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="jog">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>471</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/jog-off.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="power">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/assets/assets/power-on.png);
border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/pressed-bg-down.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/pressed-bg-up.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="incTemp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/assets/assets/temp-up-pressed.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="decTemp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
image: url(:/assets/assets/temp-down-pressed.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Temp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="text">
     <string>35</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTemp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/assets/assets/fan-4-on.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
